stdlib/public/core/StringBridge.swift

```diff
@@ -306,6 +301,30 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     return try body(selfAsPointer, rhsAsPointer)
   }
 
+  @objc
+  var fastestEncoding: UInt {
+    return _core.elementWidth == 1 ? UInt(1) : UInt(10)
```

Comment:
Foundation has an obligation to maintain the values of these; they could potentially be factored out into named private constants if desired.